comment/comment.go

```diff
@@ -89,6 +90,18 @@ func (m *GormCommentRepository) Save(ctx context.Context, comment *Comment) (*Co
 	return comment, nil
 }
 
+// Delete a single comment
+func (m *GormCommentRepository) Delete(ctx context.Context, id uuid.UUID) error {
+	tx := m.db.Delete(&Comment{ID: id})
```

Comment:
make sure to add this check before delete.

```
	if id == uuid.Nil {
		return errors.NewNotFoundError("comment", id.String())
	}
```

Else a empty id will delete everything